app/src/components/v-form/form-field.vue

```diff
@@ -39,7 +39,8 @@
 			:batch-active="batchActive"
 			:disabled="isDisabled"
 			:primary-key="primaryKey"
-			@update:model-value="emitValue($event)"
+			@update:model-value="emitValue('update:modelValue', $event)"
+			@set-field-value="emitValue('setFieldValue', $event)"
```

Comment:
Lets decouple these. The `emitValue` on `update:modelValue` should exclusively act on the raw value, not have any other arguments. We can update the set-field-value handler for a simple `@set-field-value="$emit('setFieldValue', $event)"`, and handle the destructuring of `{field, value}` upstream in v-form

Reply:
You're absolutely right, it seems I wanted to share code as much as possible, but this was too much ! Fixed in last commit